.github/workflows/developments_build.yml

```diff
@@ -71,28 +71,33 @@ jobs:
 
       - name: 1. dataloading for EDM builds
         if: steps.config.outputs.rebuild == 'yes'
+        working-directory: db-developments
         run: ./devdb.sh dataloading edm && ls -l
 
       - name: Clear cache
+        working-directory: db-developments
         run: rm -rf .library
 
       - name: 2. build dev_db ...
         if: steps.config.outputs.rebuild == 'yes'
+        working-directory: db-developments
         run: |
           ./devdb.sh build
           ./devdb.sh aggregate
 
       - name: 3. qaqc dev_db ...
         if: steps.config.outputs.rebuild == 'yes'
-        run: |
-          ./devdb.sh qaqc
+        working-directory: db-developments
+        run: ./devdb.sh qaqc
 
       - name: 4. Export ...
         if: steps.config.outputs.export == 'yes'
+        working-directory: db-developments
         run: |
           ./devdb.sh export
           ./devdb.sh upload
 
       - name: 5. Archive ...
         if: steps.config.outputs.archive == 'yes'
+        working-directory: db-developments
```

Comment:
Give this whole file a lookover - we might want to make some tweaks. I'm not entirely sure what happens in the case where "rebuild" is set to false, as there's no apparent mechanism for storage/persistence of build exports outside of this job... so if there's no rebuilding it shouldn't be possible to export. Right? So maybe can just take out that argument and simplify some of the logic around variables